controller/config.go

```diff
@@ -43,6 +43,9 @@ const (
 	// properly.
 	ControllerAPIPort = "controller-api-port"
 
+	// Canonical name for the controller
```

Comment:
It won't be used in this PR, but we need a ControllerName() getter and a unit test

Reply:
Done & Done